code/modules/mob/living/carbon/xenomorph/damage_procs.dm

```diff
@@ -119,6 +119,8 @@
 	if(stat == DEAD)
 		return FALSE
 
+	last_damage_time = world.time
```

Comment:
I know this is currently only useful for xeno's but would it make sense to move this to living. So that other things can make use of a combat timer?

Reply:
not currently no